docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
+
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/app
```

Comment:
The volume mapping `- ./:/app` might include unnecessary files in the Docker container. Consider specifying only the necessary directories or files to avoid potential issues and keep the container environment clean.